src/deploy_inst.cc

```diff
@@ -15,11 +15,11 @@ void DeployInst::Build(cyclus::Agent* parent) {
   cyclus::Institution::Build(parent);
   BuildSched::iterator it;
   std::set<std::string> protos;
+  std::string idstr = std::to_string(id());
```

Comment:
As you feed a stringstream you can probably keep an int there that should just work ;)